ariadne/utils.py

```diff
@@ -0,0 +1,260 @@
+import torch
+import numpy as np
+import itertools
+import pandas as pd
+from copy import deepcopy
+from ariadne.tracknet_v2.metrics import point_in_ellipse
+import faiss
+import matplotlib.pyplot as plt
+from scipy.interpolate import make_interp_spline, BSpline
+import glob
+import os
+
+def cartesian_product_two_stations(df):
```

Comment:
How do you think about the name like this - `brute_force_hits_two_first_stations` ?

Reply:
Changed to this name